README.md

```diff
@@ -224,7 +224,7 @@ Excluded: <Comma separated list of labels to ignore>
 [Annotations]
 Excluded: <Comma separated list of annotations to ignore>
 
-[Teams Client]
```

Comment:
Seems to be a mistake because the source code uses `Microsoft Teams Client`, in `configuration.py` we have this.

![image](https://github.com/idealista/prom2teams/assets/1327782/c28170c7-8d1d-4567-9046-80f1ac4f04f7)